### PR TITLE
Portfolio Contact section

### PR DESCRIPTION
**Pull request by DaveZag**
# Portfolio Contact Section
> ### We used [template 2](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) for this project.
## In this branch We:
- Create the HTML markup for the contact section( the form and the inputs for name, email but also the text area for a message);
- Style the __Contact section__  and its elements by using Flex-box ;
- Fix some image path errors.
- Ignore in .gitignore all unessecary files.

---
....

### CODE REVIEW

**Review by cynthiainga — COMMENTED** [review]
Hi @DaveZag, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is not appropriate to the activity, I think the appropriate activity for this project is the `contact form` but you have submitted the project for `Portfolio: desktop version: day 2`.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -326,6 +326,67 @@ <h3 class="lang-title flex">
...
+                action="https://formspree.io/f/xoqbvjnd"
+                method="POST"
+                class="flex flex-col"
+              >
+                <input type="text" name="fullname" placeholder="Full name" maxlength="30" required/>
+                <input type="email" name="email" placeholder="Email address" required/>
+                <textarea name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+
+                <div class="contact-btn flex flex-center">
+                  <button type="submit" class="button">Get in touch</button>
```
Comment (Nemwel-Boniface) [inline index.html:353]: - [ ] Great job using button states eg on hover, on click, etc. However, you used the wrong button states as per the chosen Figma design. Kindly add the correct button states to make your project look perfect and consistent with the style guides.

**Style guides for the template 2 buttons and links states:**

image

**Review by Nemwel-Boniface — CHANGES_REQUESTED** [review]
## Hello @DaveZag            :wave: , 
I'm Nemwel your reviewer for this PR :man_beard: 

You have done a great job with your portfolio footer section so far :clap: 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️  


## ♻️ Required Changes ♻️

_Check the comments under the review._

## [Optional] suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding! :partying_face: 👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (DaveZag) [reply index.html:353]: Hello @Nemwel-Boniface , 
Thanks for the feedback. The required changes concerning link states have been implemented.

...

**Review by codecaiine — APPROVED** [review]
Hi @DaveZag @Nedramrez , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
-Great UI design :heavy_check_mark: 
- Great project implementation :heavy_check_mark: 
- Good commits history :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**